docs/content.zh/docs/connectors/legacy-flink-cdc-sources/overview.md

```diff
@@ -304,4 +305,93 @@ you can construct `JsonDebeziumDeserializationSchema` as following:
       new JsonDebeziumDeserializationSchema(true, customConverterConfigs);
 ```
 
+## Building from source
+
+Prerequisites:
+- git
+- Maven
+- At least Java 8
+
+```
+git clone https://github.com/apache/flink-cdc.git
+cd flink-cdc
+mvn clean install -DskipTests
+```
+
+The dependencies are now available in your local `.m2` repository.
+
+### Code Contribute
```

Comment:
I think "Developer Guide - Contribute to Flink CDC" should have covered how to contribute to Flink CDC, and it's also not appropriate for other sections like code style and doc contribution to be here.